### PR TITLE
Phantasm

### PR DESCRIPTION
**Pull request by amanpreet692**

### CODE REVIEW

File: s2and/data.py [hunk 1]
```diff
@@ -942,7 +943,8 @@ def split_cluster_signatures_fixed(
         """
         # for now original blocks are required as it's what AMiner uses for fixed splits
         # AMiner is the only dataset that comes with its own split
-        blocks = self.get_original_blocks()
+        #blocks = self.get_original_blocks()
+        blocks = self.get_blocks()
```
Comment (sergeyf) [inline s2and/data.py:947]: Why this modification?

File: s2and/data.py [hunk 2]
```diff
@@ -195,9 +195,10 @@ def __init__(
     ):
         if mode == "train":
             if train_blocks is not None and block_type != "original":
-                raise Exception(
-                    "If you are passing in training/val/test blocks, then you probably want original blocks."
-                )
+                logger.warning("If you are passing in training/val/test blocks, then you probably want original blocks.")
```
Comment (sergeyf) [inline s2and/data.py:198]: Why this modification?

**Review by sergeyf — COMMENTED** [review]
I have a few questions.

**Comment by amanpreet692** [comment]
@mdarcy220 Could you take a look at Sergey's queries here please

**Comment by sergeyf** [comment]
Also, tests are failing! Please investigate.

**Comment by mdarcy220** [comment]
Ok this was a year ago so my memory is quite hazy, but after looking back at the code here's my guess at what's going on:

For phantasm, we have s2and, s2and-mini, and the triplet version of s2and, and we wanted to use a consistent train/dev/test split of blocks across all of them.  We really wanted to use "s2" blocks for this, because those are the ones used in the original s2and eval script (we wanted our numbers to be comparable, I think) and both s2and-mini and the s2and triplet data were constructed using "s2" blocks.

S2and doesn't allow passing custom block splits for the "s2" block type; it only allows it for "original", because in the split_cluster_signatures_fixed function it overrides the block type to "original".  I'm not sure why it's set up that way but my best guess is that long ago the only reason fixed splits were implemented was to support Aminer's fixed splits in particular, which used "original" block names.

To support custom splits for s2 blocks, we had to get rid of the exception that was raised and also get rid of the forced override to "original" blocks.  The failing test case was apparently set up to check for that exception being raised.

All things considered, I think the simplest way to resolve this is to remove the test case, unless we have a good reason to prevent people from passing custom splits for s2 block names.

**Comment by sergeyf** [comment]
I'm ok with this solution. Going to remove the test and see if it passes.

**Comment by sergeyf** [comment]
@mdarcy220 or @amanpreet692 the two new scripts are failing flake or black. Can one of you please fix?

**Comment by sergeyf** [comment]
Oh boy more fun errors

**Comment by mdarcy220** [comment]
It passed!

**Comment by sergeyf** [comment]
Bless you

**Review by sergeyf — APPROVED** [review]

**Pull request merged**